api/index.md

```diff
@@ -1,14 +1,15 @@
 ---
 layout: ApiBrowserPage
 hide_bc: true
-title: .NET API Browser
+title: .NET API browser
 quickFilterColumn1: netframework-4.8,netcore-3.1,netstandard-2.1
 quickFilterColumn2: xamarinios-10.8,xamarinandroid-7.1,xamarinmac-3.0
-quickFilterColumn3: azure-dotnet,aspnetcore-3.1,ml-dotnet
+quickFilterColumn3: azure-dotnet,aspnetcore-3.1,efcore-3.1
+quickFilterColumn4: roslyn-dotnet,ml-dotnet,spark-dotnet
```

Comment:
I've moved ML.NET to the new column because it seemed more reasonable to have asp.net core and entity framework core next to each other

Reply:
I think it only supports three columns. We could check if adding a new value in each column would work instead.

![image](https://user-images.githubusercontent.com/12971179/73095252-04d0d480-3e97-11ea-8978-6d8f5d0bcf10.png)